serving/resource_model_serving_provisioned_throughput.go

```diff
@@ -0,0 +1,116 @@
+package serving
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/databricks/databricks-sdk-go/service/serving"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const (
+	defaultPtProvisionTimeout = 10 * time.Minute
+)
+
+func ResourceModelServingProvisionedThroughput() common.Resource {
+	s := common.StructToSchema(
+		serving.CreatePtEndpointRequest{},
+		func(m map[string]*schema.Schema) map[string]*schema.Schema {
+			m["name"].ForceNew = true
+			common.MustSchemaPath(m, "config", "traffic_config").Computed = true
+			common.MustSchemaPath(m, "config", "served_entities", "name").Computed = true
+			common.MustSchemaPath(m, "config", "served_entities", "provisioned_model_units").Required = true
+			common.MustSchemaPath(m, "config", "served_entities", "entity_name").Required = true
+			common.MustSchemaPath(m, "config", "served_entities", "entity_version").Required = true
+			common.MustSchemaPath(m, "config", "served_entities", "entity_version").Optional = false
```

Comment:
let use `common.CustomizeSchemaPath(s, "<attr>").SetComputed()`, `common.CustomizeSchemaPath(s, "<attr>").SetRequired()`, etc. - it will remove the need to have changes two times like:

```
common.MustSchemaPath(m, "config", "served_entities", "entity_version").Required = true
common.MustSchemaPath(m, "config", "served_entities", "entity_version").Optional = false
```